examples/sample_dynamic_requires.py

```diff
@@ -1,7 +1,7 @@
 import luigi
 
-import gokart
-'''
+import gokart  # noqa isort:skip
```

Comment:
I couldn't understand what is this mean.
If there are common reason you add this on several files, you should add configs on `[tool.isort]` section in `pyproject.toml`